sphinx_wagtail_theme/layout.html

```diff
@@ -105,7 +106,7 @@
                 {% endif %}
                 <button class="navbar-toggler btn btn-primary d-lg-none" type="button" data-toggle="collapse" data-target="#collapseSidebar" aria-expanded="false" aria-controls="collapseExample">
                     <span class="navbar-toggler-icon"></span>
-                    <span class="sr-only">Menu</span>
+                    <span class="sr-only">{{ _('menu') }}</span>
```

Comment:
It seems odd to have this inside the toggler button. Since it is `sr-only`, would it make sense to put it directly before this button instead? Then the nasty translate/position CSS might not be necessary.

Reply:
Ah sorry, looking at the wrong thing. Disregard!